codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttConnectPayload.java

```diff
@@ -59,6 +77,10 @@ public String userName() {
     }
 
     public String password() {
```

Comment:
please deprecate this method an pointing folks to `passwordBytes()`.
also consider adding to the javadocs that this method may create a new string on each invocation.